.env

```diff
@@ -48,11 +48,19 @@ MAILER_SENDER_NAME=Kontrolgruppen
 
 DISPLAY_NO_SENSITIVE_INFORMATION_WARNING=true
 
-ROUTER_REQUEST_CONTEXT_HOST=kontrolgruppen.local.itkdev.dk
+ROUTER_REQUEST_CONTEXT_HOST=0.0.0.0
 ROUTER_REQUEST_CONTEXT_SCHEME=http
 # Optional (must start with a slash)
 #ROUTER_REQUEST_CONTEXT_BASE_URL=/some-path
 
+
+# Datafordeler URL
+DATAFORDELER_URL='http://host.docker.internal:9000/'
```

Comment:
This seems wrong. Unless I'm missing something, we should hit `http://nginx` rather that the docker host.

Reply:
This was for testing purposes locally. Since the connection for Datafordeleren was delayed tremendously, a Mock-data service for Datafordeleren was built and ran locally on the machine. Since this was a temporary solution, we decided not to add it to the docker setup, as there is a test-environment (which we still don't have access to) available. Nonetheless, we still need the environment variable and it will be replaced on deployment to Datafordeleren's api

Reply:
Ok, but then we'll have a test system where the client cannot test Datafordeleren (with neither test nor mock data).

Reply:
This is why we need to use production variables now on the test server for Datafordeleren. This release is MAINLY to test whether or not SAML still works after upgrade of PHP and Symfony, which is why we had a bunch of code uncommented